sources/engine/Stride.Graphics/GraphicsPresenter.cs

```diff
@@ -175,14 +151,29 @@ public void Resize(int width, int height, PixelFormat format)
 
             Description.BackBufferWidth = width;
             Description.BackBufferHeight = height;
-            Description.BackBufferFormat = format;
+            Description.BackBufferFormat = NormalizeBackBufferFormat(format);
 
             ResizeBackBuffer(width, height, format);
             ResizeDepthStencilBuffer(width, height, format);
 
             GraphicsDevice.End();
         }
 
+        private PixelFormat NormalizeBackBufferFormat(PixelFormat backBufferFormat)
+        {
+            // If we are creating a GraphicsPresenter with 
+            if (GraphicsDevice.Features.HasSRgb && GraphicsDevice.ColorSpace == ColorSpace.Linear)
+            {
+                // If the device support SRgb and ColorSpace is linear, we use automatically a SRgb backbuffer
+                return backBufferFormat.ToSRgb();
+            }
+            else
```

Comment:
Previous code has a condition for this case: `else if (!device.Features.HasSRgb)`
Then the default situation would be to keep `backBufferFormat` as is.

Reply:
We removed this condition on purpose because as we understood it, one either 
- has a device which is not capable -> ToNonSrgb (that was the !device.Featues.HasSRGB)
- asked for Gamma space -> ToNonSrgb
So in both cases one wants to end up with a NonSrgb format.

Reply:
OK, it makes sense, thanks (I wasn't sure if it was intentional)